package.json

```diff
@@ -8,13 +8,13 @@
     "build": "npm run build-clear && npm run build-ts && npm run build-babel",
     "lint": "tslint -p ./tsconfig.json",
     "lint:fix": "tslint -p ./tsconfig.json --fix",
-    "test:unit": "ts-node --files ./tests/unit/index.ts",
+    "test:unit": "TZ=UTC ts-node --files ./tests/unit/index.ts",
```

Comment:
What is the reason we need to force a timezone? I find it weird that we need to force a timezone, I ran the tests locally with and without the timezone and it seems to work.

Reply:
Actually now that `created` is an iso string, it's probably not required. Before the local tests for me came back with PDT and on Travis as UTC. I'll verify

Reply:
Cool, sounds good, if you are able to remove it, don't forget to remove it from "prepublish" as well